src/test/java/com/midaswebserver/midasweb/services/LoginServiceImpTest.java

```diff
@@ -0,0 +1,95 @@
+package com.midaswebserver.midasweb.services;
+
+import com.midaswebserver.midasweb.forms.LoginForm;
+import com.midaswebserver.midasweb.models.User.User;
+import com.midaswebserver.midasweb.repositories.UserRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.List;
+
+import static org.springframework.test.util.AssertionErrors.*;
+
+@SpringBootTest
+public class LoginServiceImpTest {
+    //get services and other required objects to test or to aid the testing
+    @Autowired
+    private LoginService loginService;
+    @Autowired
+    private UserRepository userRepo;
+
+    @BeforeEach
```

Comment:
Because EVERY test inserts a record into the database, it makes certain kinds of unit tests impossible to test.